dependencies.props

```diff
@@ -1,14 +1,59 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- Source of truth for dependency tooling: the commit hash of the dotnet/versions master branch as of the last auto-upgrade. -->
+
+  <!-- For detailed explanation of the reference versions see:
+
+        https://github.com/dotnet/buildtools/blob/master/Documentation/project-nuget-dependencies.md
+        https://github.com/dotnet/buildtools/blob/master/Documentation/annotated-dependency-props.md
+
+       These ref versions are pulled from https://github.com/dotnet/versions.
+  -->
   <PropertyGroup>
-       <CoreCLRVersion>2.0.0-preview2-25226-02</CoreCLRVersion>
-       <JitVersion>2.0.0-preview2-25226-02</JitVersion>
+    <CoreFxCurrentRef>f3d23289c93469d99c811fe911679d6c96ebfebf</CoreFxCurrentRef>
+    <CoreClrCurrentRef>f3d23289c93469d99c811fe911679d6c96ebfebf</CoreClrCurrentRef>
+    <StandardCurrentRef>f3d23289c93469d99c811fe911679d6c96ebfebf</StandardCurrentRef>
+  </PropertyGroup>
+  
+  <PropertyGroup>
+    <PlatformPackageVersion>2.0.0-preview2-25226-02</PlatformPackageVersion>
+    <CoreCLRVersion>2.0.0-preview2-25226-02</CoreCLRVersion>
+    <JitVersion>$(CoreCLRVersion)</JitVersion>
```

Comment:
If CoreCLR does a servicing release that only includes one of these, they'll start to version separately and it would help to have them upgraded individually--but I don't think that ever happens, so this is probably fine. /cc @gkhanna79